solution/backend/resources/views/resources.py

```diff
@@ -61,29 +62,39 @@ def get_annotated_url(self):
             default=None,
         )
 
-    def format_gov_results(self, gov_results):
-        if "errors" in gov_results or (gov_results["web"]["total"] == 0 and not gov_results["web"]["results"]):
-            raise NotFound("Invalid page")
+    def format_gov_results(self, gov_response):
         results = []
-        for gov_result in gov_results['web']['results']:
-            results.append({
-                'name': gov_result['title'],
-                'snippet': gov_result['snippet'],
-                'url': gov_result['url']
-            })
-        self.gov_results = {
-            "total": gov_results["web"]["total"],
-            "results": results,
-        }
+        if "errors" in gov_response or (gov_response["web"]["total"] == 0 and not gov_response["web"]["results"]):
+            self.gov_results = {
+                'total': 0,
+                'results': results
+            }
```

Comment:
Can we put something in the logs that there was an error? That way should we notice that we are switching to our metadata we can look at the logs and see exactly what the issue was.

Reply:
We do not currently have logging enabled on our django application and in order to do that properly is out of the scope of this ticket.  if we want to do logging, we should create another ticket.